SwissDataScienceCenter/renku-graph
# fix: so new events are added to existing batch if under processing

**Testing scenario:**
1. log in to https://renku-kuba.dev.renku.ch;
2. chose a project you have rights to push;
3. checkout the project either locally or in a notebook;
4. create few new commits for the project without pushing them to the remote (~10 commits should be enough but they can be for simple changes like README updates - I repeated the following command 10 times: `echo test >> README.md && git add . && git commit -m "updated readme"`);
5. do a single push for all the commits and observe the progress bar on e.g. the Project Datasets page;
6. do another series of commits and do a single push;
7. the way the progress bar should react on the new push is that the progress should go back and show a lower percentage towards completion;

**NOTICE:** points 5, 6 and 7 have to be done relatively quickly so the first series of events don't get all processed before the second series. The easiest but a bit more time-consuming approach would be to generate a bigger number of commits in the first series.

**Image for testing:**
```
jachro/webhook-service:bc8beb7e
```

closes #283
